lib/ClangImporter/ClangImporter.cpp

```diff
@@ -8348,3 +8355,63 @@ AccessLevel importer::convertClangAccess(clang::AccessSpecifier access) {
     return AccessLevel::Public;
   }
 }
+
+ClangInheritanceInfo
+ClangInheritanceInfo::forUsingDecl(const clang::UsingShadowDecl *usingDecl) {
+  ClangInheritanceInfo info;
+
+  auto *derivedRecord =
+      llvm::cast_if_present<clang::CXXRecordDecl>(usingDecl->getDeclContext());
```

Comment:
Do you need the `llvm::` for the cast? If you do, I wonder if we should change this as we already access `isa` and similar cast functions without having to spell the namespace out.

Reply:
Ah I had forgotten that Swift has declarations like `using llvm::isa` and `using llvm::dyn_cast_or_null`. I just switched to using the latter.

I had originally used the former because the documentation for `llvm::dyn_cast_or_null` suggests that it is deprecated in favor of `llvm::cast_if_present`.